static/templates/settings/deactivated_users_admin.hbs

```diff
@@ -13,7 +13,7 @@
         <table class="table table-condensed table-striped wrapped-table">
             <thead class="table-sticky-headers">
                 <th class="active" data-sort="alphabetic" data-sort-prop="full_name">{{t "Name" }}</th>
-                <th {{#if can_sort_by_email}}data-sort="email"{{/if}}>{{t "Email" }}</th>
+                <th {{#if allow_sorting_deactivated_users_list_by_email}}data-sort="email"{{/if}}>{{t "Email" }}</th>
```

Comment:
Why does this need a different structure than what we have in the normal users list?

Reply:
For normal users list, there will always be atleast one email (that of the user itself) which will be visible and thus we allow sorting.

Reply:
OK -- I think what I'd expect is for the `data-sort` hover option to be available even with an empty table. But probably there's no strong reason to remove this logic if you've already got it "working".